lib/myads/my_banner_ads.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:google_mobile_ads/google_mobile_ads.dart';
+
+class MyBannerAds {
```

Comment:
name this class as EasyAdmobBannerAd

Reply:
create an abstract class for this:

EasyAd

- Future init()
- dynamic show
- void dispose
- Future load
- bool get isAdLoaded
- VoidCallback onAdLoaded
- VoidCallback onAdDisplayed
- VoidCallback onAdDismissed
- VoidCallback onAdCompleted

Reply:
I will work on it 👍